boites/views.py

```diff
@@ -40,6 +40,40 @@ def json_view(request, api_key):
     boite.save()
     return JsonResponse(boite.get_apps_dictionary(), safe=False)
 
+def create_app_view(request, pk):
+    boite = get_object_or_404(Boite, pk=pk)
```

Comment:
you should add `user=request.user` in `get_object_or_404` to ensure only the owner can access this view